addons/html_editor/i18n/html_editor.pot

```diff
@@ -1793,7 +1799,7 @@ msgstr ""
 #. module: html_editor
 #. odoo-javascript
 #: code:addons/html_editor/static/src/main/media/image_plugin.js:0
-msgid "Transform the picture (click twice to reset transformation)"
+msgid "Transform the picture"
```

Comment:
If we do that in 18.0, we will break all translations for that term. If we can keep the behavior of clicking twice to reset the transformation, we could leave this term as is, which would be better.